packages/data-manipulation/html/README.md

```
@@ -0,0 +1,508 @@
+// Result: '< >' (entities without semicolon replaced when not followed by =)
+
+// Decode with strict scope - ignores entities without semicolon
+decode("&lt &gt", { scope: "strict" });
+// Result: '&lt &gt' (entities without semicolon ignored)
+```
+
+**Decode Options:**
+
+- `level`: `'all'` (alias to `'html5'`) | `'html5'` (default) | `'html4'` | `'xml'` - Specifies the standard to use for named character references
+- `scope`: `'body'` (default) | `'attribute'` | `'strict'` - Controls how entities without semicolons are handled
+    - `'body'`: Emulates browser behavior when parsing tag bodies - entities without semicolon are also replaced
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)